modules/react-native-login/options.js

```diff
@@ -1,29 +1,37 @@
-const LOGO_URL =
-  "https://crowdbotics-slack-dev.s3.amazonaws.com/media/project_component_resources/cb-icon.png";
-const BACKGROUND_URL =
-  "https://crowdbotics-slack-dev.s3.amazonaws.com/media/project_component_resources/halfbg.png";
+/*
+Login Module Options
 
-// Change this the  name of the screen you want to redirect after a succesful login.
-const HOME_SCREEN_NAME = "Home Screen";
+You can use either this file or the Module Options UI, found 
+in the Crowdbotics Storyboard by clicking on "Properties" of
+the module after it has been installed.
 
-const validateEmail = /^[^\s]+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+Here is an explanation of these values:
 
-//These two options control the what the navigation tabs say, keep them short!
-const SignInNavText = "Sign In";
-const SignUpNavText = "Sign Up";
+LOGO_URL: Set a Logo image for your login screen. 
+For best results, use a similar size to the sample provided. 
 
-//These two options control your button text.
-const SignInButtonText = "Login";
-const SignUpButtonText ="Sign Up";
+BACKGROUND_URL: Set a Background image for your login screen. 
+For best results, use a similar size to the sample provided. 
 
+HOME_SCREEN_NAME: Use this to set the name of the screen to redirect to after 
+a succesful login. 
+
+validateEmail: This Regex expression can be customized for additional email
+validation logic.
+
+SignInNavText/SignUpNavText: These two options control the what the navigation tabs say, keep them short!
+
+SignInButtonText/SignUpButtonText: These two options control your button text.
+
+*/
 
 export default {
-  LOGO_URL: LOGO_URL,
-  BACKGROUND_URL: BACKGROUND_URL,
-  HOME_SCREEN_NAME: HOME_SCREEN_NAME,
-  validateEmail: validateEmail,
-  SignInNavText: SignInNavText,
-  SignUpNavText: SignUpNavText,
-  SignInButtonText: SignInButtonText,
-  SignUpButtonText: SignUpButtonText,
+  LOGO_URL: "https://crowdbotics-slack-dev.s3.amazonaws.com/media/project_component_resources/cb-icon.png",
+  BACKGROUND_URL: "https://crowdbotics-slack-dev.s3.amazonaws.com/media/project_component_resources/halfbg.png",
+  HOME_SCREEN_NAME: "Home Screen",
+  validateEmail: /^[^\s]+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/,
```

Comment:
The parser doesn't support nodes of the type `RegExpLiteral`.

Reply:
https://astexplorer.net/#/gist/cc99dbdbea3788540fa30346b129472f/a57e11fe75bd82fbe79eb0e7d2b44bf22e33420b